cmd/machine-config-daemon/start.go

```diff
@@ -32,6 +32,7 @@ var (
 		fromIgnition           bool
 		kubeletHealthzEnabled  bool
 		kubeletHealthzEndpoint string
+		metricsURL             string
```

Comment:
Q: metricsURL or promMetricsURL? i dont know, you tell me, is this generic or specific only for prom?

Reply:
This is technically specific to prom since the metricsURL will have a prom HTTP handler.  Will update.